hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestBase.java

```diff
@@ -63,6 +64,9 @@
   protected static final String HUDI_UTILITIES_BUNDLE =
       HOODIE_WS_ROOT + "/docker/hoodie/hadoop/hive_base/target/hoodie-utilities.jar";
   protected static final String HIVE_SERVER_JDBC_URL = "jdbc:hive2://hiveserver:10000";
+  protected static final String PRESTO_COORDINATOR_URL = "presto-coordinator-1:8090";
+  protected static final String PRESTO_USE_HIVE_DEFAULT = "use hive.default;";
```

Comment:
remove this line ? Seems like you are already passing catalog and schema to the presto command line